src/main/java/org/broadinstitute/hellbender/tools/walkers/bqsr/ApplyBQSR.java

```diff
@@ -45,10 +45,10 @@
  *
  * <h3>Usage example</h3>
  * <pre>
- * ./gatk-launch ApplyBQSR \
+ * ./gatk ApplyBQSR \
```

Comment:
Update program group, see my comments to ApplyBQSRSpark and update the example command. For example, notice how I removed the reference from the ApplyBQSRSpark command. Example commands should represent the most straightforward way to run the tool and (hopefully) encompass the most typical use-cases.

Reply:
Fixed